client/verta/verta/_cli/registry/create.py

```diff
@@ -28,14 +30,28 @@ def create():
 @click.option("--label", "-l", multiple=True, help="Labels to be associated with the object.")
 @click.option("--visibility", "-v", default="private", show_default=True, type=click.Choice(["private", "org"], case_sensitive=False), help="Visibility level of the object.")
 @click.option("--workspace", "-w", help="Workspace to use.")
-def create_model(model_name, label, visibility, workspace, description):
+@click.option("--action_type", "-a", default="other", show_default=True, type=click.Choice(["other", "classification", "clustering", "detection", "regression", "transcription", "translation"], case_sensitive=False), help="Action type.")
+@click.option("--data_type", "-d", default="other", show_default=True, type=click.Choice(["other", "audio", "image", "tabular", "text", "video"], case_sensitive=False), help="Data type.")
```

Comment:
`-d` is already taken by `--description`. Maybe we can just leave those out for now 😬 

Also, for CLI option names, `-` is preferred over `_`.

```suggestion
@click.option("--action-type", default="other", show_default=True, type=click.Choice(["other", "classification", "clustering", "detection", "regression", "transcription", "translation"], case_sensitive=False), help="Action type.")
@click.option("--data-type", default="other", show_default=True, type=click.Choice(["other", "audio", "image", "tabular", "text", "video"], case_sensitive=False), help="Data type.")
```